htdocs/includes/db_lib.php

```diff
@@ -5817,9 +5817,21 @@ function add_user($user)
 		if($user->level == 17) {
 			$user->rwoptions = LabConfig::getDoctorUserOptions();
 		}
+		// If level = 20, generate the satellite_lab_id value
+        if ($user->level == 20) {
```

Comment:
```suggestion
        if ($user->level == $LIS_SATELLITE_LAB_USER) {
```

Since this is defined as a constant, would be better to use it that way.